api/app/src/models/medical/location-address.ts

```diff
@@ -0,0 +1,4 @@
+import { Address } from "./address";
+
+export type LocationAddress = Pick<Address, "addressLine2"> &
```

Comment:
The thought behind `LocationAddress` was like 'PhysicalLocationAddress' in the sense of a building location rather than a personal address -> agreed `AddressStrict` is easier to grok though so will update